app.ts

```diff
@@ -196,6 +202,37 @@ setInterval(async () => {
       let memberData = memberDoc.data() as MemberData
       if (memberData[date].timeStamp === undefined) {
         let reporterName = memberData[date].reporter
+        const optKeyboard: telegramBot.SendMessageOptions = {
+          parse_mode: "Markdown",
+          reply_markup: {
+            keyboard: [[{ text: "Source Did not respond yet" }], [{ text: "Did not asked the report yet" }]]
```

Comment:
Second message could be "Did not ask the source yet" for better clarity. Feel free to suggest a different sentence

Reply:
Cool 🙌